src/intTest/java/uk/nhs/digital/nhsconnect/nhais/controller/FhirControllerIntegrationTest.java

```diff
@@ -0,0 +1,135 @@
+package uk.nhs.digital.nhsconnect.nhais.controller;
+
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.core.io.Resource;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.web.servlet.MockMvc;
+import uk.nhs.digital.nhsconnect.nhais.IntegrationTestsExtension;
+import uk.nhs.digital.nhsconnect.nhais.repository.OutboundStateRepository;
+import java.nio.file.Files;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.header;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+@ExtendWith({SpringExtension.class, IntegrationTestsExtension.class})
+@SpringBootTest
+@AutoConfigureMockMvc
+@Slf4j
+public class FhirControllerIntegrationTest {
+    private static final String expectedOperationId = "c35b1432682a04ca77b287b11f240fbd8c46f22fe589b513a84e307b57dcb820";
+
+    @Autowired
+    OutboundStateRepository outboundStateRepository;
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Value("classpath:patient/validAcceptanceInput.json")
+    private Resource validAcceptancePayload;
+
+    @Value("classpath:patient/not-json.xml")
+    private Resource notJsonPayload;
+
+    @Value("classpath:patient/deduction-missing-destination-ha-cipher.fhir.json")
+    private Resource deductionNoHaCipher;
+
+    @Value("classpath:patient/deduction-missing-gp-code.fhir.json")
+    private Resource deductionNoGpCode;
+
+    @Value("classpath:patient/deduction-missing-gp-trading-partner-code.fhir.json")
+    private Resource deductionNoTradingPartnerCode;
+
+    @Value("classpath:patient/removal-missing-destination-ha-cipher.fhir.json")
+    private Resource removalNoHaCipher;
+
+    @Value("classpath:patient/removal-missing-gp-code.fhir.json")
+    private Resource removalNoGpCode;
+
+    @Value("classpath:patient/removal-missing-gp-trading-partner-code.fhir.json")
+    private Resource removalNoTradingPartnerCode;
+
+    @Value("classpath:patient/deduction-invalid-json-structure.fhir.json")
+    private Resource deductionInvalidJsonStructure;
+
+    @Value("classpath:patient/removal-invalid-json-structure.fhir.json")
+    private Resource removalInvalidJsonStructure;
+
+    @Test
+    void whenValidAcceptanceInput_thenReturns202() throws Exception {
```

Comment:
We can remove this scenario. It is covered by UAT tests. One less example file to maintain.

Reply:
ok